src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -10,16 +9,16 @@
 import os
 import platform
 import re
-import stat
 import shutil
+import stat
```

Comment:
add some perf number in PR description

Reply:
e2e perf gain will be got from vs extension later

Reply:
number updated